Library/Formula/gnuradio.rb

```diff
@@ -0,0 +1,230 @@
+class Gnuradio < Formula
+  desc "SDK providing the signal processing runtime and processing blocks"
+  homepage "https://gnuradio.org/"
+
+  stable do
+    url "http://gnuradio.org/releases/gnuradio/gnuradio-3.7.8rc1.tar.gz"
+    sha256 "a2228c8cba0194924c7549e37085b5c3837c3282ac96f5bf2392c07cbaed0895"
+  end
+
+  # cheetah starts here
+  resource "Markdown" do
```

Comment:
`resource` blocks should go beneath the `option` and `depends_on` blocks. Thanks!